### ISSUE TITLE
finetune get worse result

### ISSUE THREAD

**Issue by DENGBOYU-REX** [advimman/lama#342]
Hi, thanks for your great work! I am trying to fine-tune the big-lama model on my custom dataset, but the output after fine-tuning seems worse than the original version.

Here is the original version output:
![19_image](https://github.com/user-attachments/assets/11ac6c09-c070-426b-9710-223d84f3f2d9)
![19_inpaint](https://github.com/user-attachments/assets/57b8cd8d-9283-4a15-a1ce-00f8c46912e8)
![19_mask](https://github.com/user-attachments/assets/bc95dcba-d3c4-4e08-8716-ec19ad48051e)
And here is the fine-tuned output:
![19_image](https://github.com/user-attachments/assets/c5772ac6-9b08-4b4f-8638-6fa8034adc62)
![19_inpaint](https://github.com/user-attachments/assets/8b3c21a0-5aea-4561-a62b-9e2ccf617583)
![19_mask](https://github.com/user-attachments/assets/45cfd464-7522-49d5-8b50-77df5d0b3747)
I fine-tuned the model at 512x512 resolution. Could this be the issue? And i just use perceptual loss from ResNet50 and MSE loss. I would appreciate any advice you can provide!

**Comment by dingluo1102** [comment]
Hello! I'd like to know how to change lama version(such as fourier, big lama etc.)
